src/ViewHelper/Capturer.php

```diff
@@ -40,27 +29,19 @@ public function __invoke(string $contentKey = null): AbstractViewHelper
         return $this;
     }
 
-    /**
-     * @return string
-     */
     public function getContentKey(): string
     {
         return $this->activeContentKey;
     }
 
-    /**
-     * Start capturing.
-     */
     public function start(): void
     {
         // Start capturing
         \ob_start();
     }
 
     /**
-     * End capturing.
-     *
-     * @return Capturer|AbstractViewHelper
+     * @throws \LogicException
```

Comment:
Not sure if `\LogicException` is something that generally should be noted in the PHPDocs... 🤔 
A `\LogicException` should usually never be caught since it shows that something has gone very wrong in the code flow (for example using a method before the object is completely instantiated)

Reply:
agreed!